onnxruntime/core/graph/graph.cc

```diff
@@ -4213,7 +4200,9 @@ Status Graph::LoadFromOrtFormat(const onnxruntime::fbs::Graph& fbs_graph,
 #if !defined(ORT_MINIMAL_BUILD)
                                   schema_registry,
 #endif
-                                  nullptr, nullptr, logger);
+                                  nullptr, nullptr, logger,
+                                  // TODO: support true in this case maybe?
+                                  false);
```

Comment:
Not sure it's meaningful to use 'true' here as the ORT format model is created via ORT from an ONNX model.

Reply:
Updated comment.